src/mappings/utils/nativeCurrency.ts

```diff
@@ -0,0 +1,8 @@
+import { Token } from "../../model";
+
+export function getNativeCurrency(): Token.KINT | Token.INTR {
+    if (process.env.SS58_CODEC === "kintsugi") {
+        return Token.KINT;
+    }
+    return Token.INTR;
```

Comment:
I guess it's possible for the native currency to be something else but we probably don't want to throw an error here, also we are only supporting Interlay and Kintsugi anyway.

Reply:
It is something that we feasibly could look up on chain, but I decided to keep it simple as I don't see us introducing a third native token for which we'd want to deploy this very same squid anytime soon.
Famous last words and all.